storage_service/locations/api/resources.py

```diff
@@ -31,7 +31,7 @@
 from locations.api.sword import views as sword_views
 
 from ..models import (Callback, CallbackError, Event, File, Package, Location, Space, Pipeline, StorageException)
-from ..forms import LocationForm, SpaceForm
```

Comment:
I left the "unused" form imports here because they're used in commented out validation code ([example](https://github.com/artefactual/archivematica-storage-service/blob/stable/0.9.x/storage_service/locations/api/resources.py#L241)) that I left as an example of how it could work. See [this]( https://github.com/artefactual/archivematica-storage-service/blob/stable/0.9.x/storage_service/locations/api/resources.py#L40-L45) for an explanation of the bug.